draft-irtf-qirg-principles-03.xml

```diff
@@ -795,6 +820,47 @@
            information via separate classical channels which the repeaters will
            have to correlate with the qubits stored in their memory.</t>
 
+           <t> The first quantum networks may not be "routing and forwarding"
+           networks.
+           <vspace blankLines="1" />
+
+           Quantum links provide Bell pairs that are undirected network
+           resources, in contrast to directed links of classical networks.
+           This phenomenological distinction should result in architectural
+           differences between quantum networks and classical networks.
+           <vspace blankLines="1" />
+
```

Comment:
If routing is the same, why not leave it out? Only contrast the data plane activity. Otherwise, I was reading expecting some explanation of how routing will be different, but this is not the point of this paragraph.